app/controllers/tags_controller.rb

```diff
@@ -94,6 +94,26 @@ def select_merge; end
 
   def merge
     @primary = @tag
+
+    # No merge to self.
+    if params[:merge_with_id].eql? @primary.id.to_s
+      flash[:danger] = 'Cannot merge a tag with itself.'
+
+      # redirect_to tag_path(id: @category.id, tag_id: @primary.id) && return
```

Comment:
This didn't do what I expected, at all.

Reply:
applying technique https://github.com/codidact/qpixel/pull/178/commits/dfae8b9f0a486aca417804fbf14e2d44308841da uses fixes it, but https://github.com/codidact/qpixel/pull/176#discussion_r487302984 is actually better